packages/react/src/jsx/ReactJSXElement.js

```diff
@@ -622,12 +617,7 @@ export function createElement(type, config, children) {
           "it's defined in, or you might have mixed up default and named imports.";
       }
 
-      const sourceInfo = getSourceInfoErrorAddendumForProps(config);
-      if (sourceInfo) {
-        info += sourceInfo;
-      } else {
-        info += getDeclarationErrorAddendum();
-      }
+      info += getDeclarationErrorAddendum();
```

Comment:
@sebmarkbage I assume I can delete this too? All it does is print "Check the render method of <name of currently rendering component>" which is some 2017 bullshit anyway.

Reply:
Yea in this case you have the full stack trace.

In the key warning it’s nice to have the owner of the child rather than the parent where the warning happens.

Reply:
anticipated my next question :) Thanks